pyproject.toml

```
@@ -63,7 +63,7 @@ Changelog = "https://ethereum.github.io/execution-spec-tests/main/CHANGELOG/"
 [project.optional-dependencies]
 test = ["pytest-cov>=4.1.0,<5"]
 lint = [
-    "ruff==0.9.4",
+    "ruff>=0.9.4",
```

Good idea, I'd also clamp the max version to avoid suprises:
```suggestion
    "ruff>=0.9.4,<1",
```